proto/lnd.proto

```diff
@@ -341,6 +341,13 @@ service Lightning {
     */
     rpc SubscribeInvoices (InvoiceSubscription) returns (stream Invoice);
 
+    /* lncli: `deletecanceledinvoice`
+    DeleteCanceledInvoice removes a canceled invoice from the database. If the
+    invoice is not in the canceled state, an error will be returned.
+    */
+    rpc DeleteCanceledInvoice (DelCanceledInvoiceReq)
```

Comment:
It took me a while to understand this commit: 4e52894d9cb040e0fae21d30052c23caea9e9f0d

The confusion stems from the description: regenerate protos

I didn't understand where the new endpoints where coming from if you are just regenerating protos. Obviously they come from the current state of the protos in the `lnd` and `loop` projects. Maybe explain that in the commit message?

`Copying in latest version of proto-files from lnd and loop and regenerating protos.`